crowdfunding_compassion/models/crowdfunding_participant.py

```diff
@@ -35,6 +35,31 @@ class CrowdfundingParticipant(models.Model):
     instagram_url = fields.Char(string="Instagram link")
     personal_web_page_url = fields.Char(string="Personal web page")
     profile_photo = fields.Binary(string="Profile photo")
+    sponsorship_url = fields.Char(compute="_compute_sponsorship_url")
+
+    @api.model
+    def create(self, vals):
+        res = super().create(vals)
+        self.env['utm.source'].create({
+            'name': res.partner_id.name
+        })
+        return res
```

Comment:
You should handle this with `_inherits` as it was suggested for crowdfunding.project